src/Microsoft.Health.Fhir.SqlServer/Microsoft.Health.Fhir.SqlServer.csproj

```diff
@@ -20,18 +20,23 @@
     <EmbeddedResource Include="Features\Schema\Migrations\6.diff.sql" />
     <EmbeddedResource Include="Features\Schema\Migrations\6.sql" />
     <GenerateFilesInputs Include="Features\Schema\Migrations\6.sql" />
+    <Generated Include="Features\Schema\Model\ImmutableTypes.Generated.cs">
+      <Generator>ImmutableSqlModelGenerator</Generator>
+      <Namespace>Microsoft.Health.Fhir.SqlServer.Features.Schema.Model</Namespace>
+      <Args>"$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\Features\Schema\Migrations\6.sql'))"</Args>
+    </Generated>
     <Generated Include="Features\Schema\Model\VLatest.Generated.cs">
-      <Generator>SqlModelGenerator</Generator>
+      <Generator>MutableSqlModelGenerator</Generator>
       <Namespace>Microsoft.Health.Fhir.SqlServer.Features.Schema.Model</Namespace>
-      <Args>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\Features\Schema\Migrations\6.sql'))</Args>
+      <Args>"$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\Features\Schema\Migrations\6.sql'))"</Args>
```

Comment:
Would it make sense to now create this path in an msbuild variable to keep these in sync?

Reply:
Reduced a lot of the duplication [here](https://github.com/microsoft/fhir-server/pull/1491/commits/c5f7c2b18b7973e8861c48ea46988b810f210316)